dspace-spring-rest/src/main/java/org/dspace/app/rest/model/RelationshipRestWrapper.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.dspace.app.rest.RelationshipRestController;
+
+public class RelationshipRestWrapper implements RestAddressableModel {
```

Comment:
Javadocs for class and all its methods

Reply:
The JavaDocs have been included for the class now
For the methods, these are basic getters and setters, for which it seems no JavaDoc has been requested for any sibling subclass of RestAddressableModel (e.g. BitstreamRest)